bvm/ballerina-core/src/main/java/org/ballerinalang/BLangProgramRunner.java

```diff
@@ -86,6 +79,34 @@ public static void runService(ProgramFile programFile) {
         }
     }
 
+    private static void deployTransactionCoordinatorServices(ProgramFile programFile, Context bContext) {
+        PackageInfo coordinatorPkgInfo = programFile.getPackageInfo("ballerina.transactions.coordinator");
```

Comment:
Shall we use "COORDINATOR_PACKAGE" string constant declared in TransactionConstants.java ?